bin/ccs-push

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+
+USAGE_MSG="usage: $0 -h hostname -c camera -s subdir -f filename [-d]"
+
+DBB_FLAG=0
+while getopts c:df:h:s: option
+do
+case "${option}"
+in
+c)
+    if [ "$OPTARG" = "auxtel" ] || [ "$OPTARG" = "comcam" ]; then
+        CAMERA=${OPTARG}
+    else
+        echo "-c argument must be auxtel or comcam"
+        exit 1
+    fi;;
+d)
+    DBB_FLAG=1
+    ;;
+f)
+    FILENAME=${OPTARG}
+    ;;
+h)
+    OODS_HOSTNAME=${OPTARG}
+    ;;
+s)
+    SUB_DIR=${OPTARG}
+    ;;
+esac
+done
+
+if [[ -z "$OODS_HOSTNAME" ]] || [[ -z "$FILENAME" ]] || [[ -z "$OODS_HOSTNAME" ]] || [[ -z "$SUB_DIR" ]]; then
+    echo $USAGE_MSG
+    exit 1
+fi
+
+if [ "$CAMERA" = "auxtel" ]; then
+    DBB_DIR=dbb
+fi;
+
+if [ "$CAMERA" = "comcam" ]; then
+    DBB_DIR=cc_dbb
+fi;
+
+OODS_LOGIN=saluser@$OODS_HOSTNAME
+STAGING_ROOT=/data/staging/$CAMERA/camera/$SUB_DIR
+OODS_LANDING_ROOT=/data/staging/$CAMERA/oods/$SUB_DIR
+DBB_LANDING_ROOT=/data/staging/$DBB_DIR/$SUB_DIR
+
+if [ "$DBB_FLAG" -eq 1 ]; then
+    ssh $OODS_LOGIN "\
+        mkdir -p $STAGING_ROOT && \
+        mkdir -p $OODS_LANDING_ROOT && \
+        mkdir -p $DBB_LANDING_ROOT && \
+        cat > $STAGING_ROOT/$FILENAME && \
+        ln $STAGING_ROOT/$FILENAME $OODS_LANDING_ROOT/$FILENAME && \
+        ln $STAGING_ROOT/$FILENAME $DBB_LANDING_ROOT/$FILENAME && \
+        rm $STAGING_ROOT/$FILENAME" <$FILENAME
```

Comment:
I would maybe do this by assembling pieces to highlight the similarities of the two commands, but this is OK.

Reply:
You know, I thought of that, but then thought I'd rather have the full commands listed to make it clearer about what was going on.   Also, the DBBBM won't always be around (I believe) if we switch it's tasks to Rucio, that portion of the code might not be around that long.